File: 2024.1-Exercícios/Aula-6/questao1.c
/* Escreva um programa que conta o número de palavras em uma string fornecida pelo usuário. Palavras são separadas por espaços em branco. */

#include <stdio.h>
#include <string.h>

int main() {
    char str[100];
    int i, count = 0;

    printf("Digite uma string: ");
    gets(str);

    for (i = 0; str[i] != '\0'; i++) {
        if (str[i] == ' ') {
            count++;
        }
    }

    printf("Número de palavras na string: %d\n", count + 1);

    return 0;
}

File: 2024.1-Exercícios/Aula-14/questao1.py
""" Escreva um programa em Python que solicite ao usuário informações de N alunos, onde N é um número informado pelo próprio usuário. Para cada aluno, o programa deve pedir o nome, a idade e a nota final. As informações devem ser salvas em um arquivo chamado alunos.txt, com o seguinte formato:
Nome: <nome>
Idade: <idade>
Nota: <nota>
---
Cada aluno deve ser separado por uma linha com ---.
Exemplo do arquivo alunos.txt criado:
Nome: Alex
Idade: 21
Nota: 8.5
---
Nome: Bob
Idade: 23
Nota: 7.2
---"""

n = int(input("Digite o número de alunos: "))
with open("alunos.txt", "w") as file:
    for i in range(n):
        nome = input("Digite o nome do aluno: ")
        idade = input("Digite a idade do aluno: ")
        nota = input("Digite a nota do aluno: ")
        file.write(f"Nome: {nome}\nIdade: {idade}\nNota: {nota}\n---\n")

print("Arquivo criado com sucesso!")